sdk/batch/microsoft-azure-batch/src/test/java/com/microsoft/azure/batch/FileTests.java

```diff
@@ -87,10 +87,10 @@ public String call(InputStream input) {
                 Assert.assertEquals("hello\n", output);
 
                 //Running this check temporarily in Record mode only, playback mode parses incorrect value from the recording.
-                if(isRecordMode()) {
-                    FileProperties properties = batchClient.fileOperations().getFilePropertiesFromTask(jobId, taskId, "stdout.txt");
-                    Assert.assertEquals(6, properties.contentLength());
-                }
+                //if(isRecordMode()) {
```

Comment:
Why is this (and line 158 below) commented out?

Reply:
That was accidentally committed--will update now